data/json/items/fake.json

```diff
@@ -12,6 +12,7 @@
         "copy-from": "fake_item",
         "type": "TOOL",
         "name": "integrated toolset",
+        "flags": "TRADER_AVOID",
```

Comment:
I simply added the flag to all items that were in the original list, to keep exact the same behavior as before. Adding the flag to more items is certainly desirable, but that is for somebody else to do.